content/news/2023.md

```diff
@@ -54,14 +54,44 @@ More info...
 -->
 
 <!-- Months always are title level 2 with ## before -->
+## December 2023
```

Comment:
January 2024?

Reply:
I thought about this too, but I would also like to have opinion of @RiffVentura